build.gradle

```diff
@@ -33,11 +33,11 @@ buildscript {
 
         // App dependencies
         androidGradlePluginVersion = '3.2.0-beta04'
-        appcompatVersion = '1.0.0-beta01'
+        appcompatVersion = '1.0.0-rc02'
         browserVersion = '1.0.0-beta01'
-        constraintLayoutVersion = '1.1.2'
-        crashlyticsVersion = '2.9.4'
-        dagger = "2.16"
+        constraintLayoutVersion = '1.1.3'
+        crashlyticsVersion = '2.9.5'
+        daggerVersion = "2.16"
```

Comment:
Could not upgrade to 2.17--ran into https://github.com/google/dagger/issues/1245